Description: Use FreeMono.ttf instead on the non-free MyUnderwood.ttf.
Author: Charlie Smotherman <cjsmo@cableone.net>  

Index: ampache-3.6-git408e713+dfsg/modules/captcha/captcha.php
===================================================================
--- ampache-3.6-git408e713+dfsg.orig/modules/captcha/captcha.php	2012-06-02 13:15:47.000000000 -0500
+++ ampache-3.6-git408e713+dfsg/modules/captcha/captcha.php	2012-06-02 21:41:46.000000000 -0500
@@ -475,7 +475,7 @@
 
    #-- return a single .ttf font filename
    function font() {
-      $fonts = array(/*"MyUnderwood.ttf"*/);
+      $fonts = array(/*"FreeMono.ttf"*/);
       $fonts += glob(CAPTCHA_FONT_DIR."/*.ttf");
       return $fonts[rand(0,count($fonts)-1)];
    }
@@ -1258,4 +1258,4 @@
 
 
 
-?>
\ No newline at end of file
+?>
